docs/data-sources/vbs_backup_policy.md

```diff
@@ -27,19 +27,19 @@ data "huaweicloud_vbs_backup_policy" "policies" {
 The arguments of this data source act as filters for querying the available VBS backup policy.
 The given filters must match exactly one VBS backup policy whose data will be exported as attributes.
 
-* `region` - (Optional) The region in which to obtain the VBS backup policy. If omitted, the provider-level region will be used.
+* `region` - (Optional, String) The region in which to obtain the VBS backup policy. If omitted, the provider-level region will be used.
 
-* `id` (Optional) - The ID of the specific VBS backup policy to retrieve.
+* `id` - (Optional, String) The ID of the specific VBS backup policy to retrieve.
 
-* `name` (Optional) - The name of the specific VBS backup policy to retrieve.
+* `name` - (Optional, String) The name of the specific VBS backup policy to retrieve.
 
-* `status` (Optional) - The status of the specific VBS backup policy to retrieve. The values can be ON or OFF
+* `status` - (Optional, String) The status of the specific VBS backup policy to retrieve. The values can be ON or OFF
 
-**filter_tags** **- (Optional)** Represents the list of tags. Backup policy with these tags will be filtered.
+**filter_tags** **- (Optional, List)** Represents the list of tags. Backup policy with these tags will be filtered.
```

Comment:
update this to * `filter_tags` - () like other arguments

Reply:
fixed